vignettes/EIRprevmatch.Rmd

```diff
@@ -15,68 +15,159 @@ knitr::opts_chunk$set(
+
+The first and fastest method uses the `malariaEquilibrium` package function `human_equilibrium()` to calculate the equilibrium *Pf*PR~2-10~ for a given EIR using the canonical equilibrium solution for the `malariasimulation` model. The second method involves running `malariasimulation` simulations across a small set of initial EIR values, extracting the *Pf*PR~2-10~ from each simulation run, and fitting a model relating initial EIR to *Pf*PR~2-10~ to allow users to predict the initial EIR value required to yield the desired *Pf*PR~2-10~. The third approach is to calibrate the model using the `cali` package function (see https://github.com/mrc-ide/cali for more information) `calibrate()` , which searches a user-defined EIR parameter space and identifies the value which yields the target *Pf*PR~2-10~ to a defined tolerance.
+
+The code chunks demonstrating the latter two methods call functions from the `mgcv` and `cali` packages. To reduce the number of dependencies when installing the `malariasimulation` package we have commented out the lines loading these packages and instructed these chunks not to run. However, users wishing to run and/or experiment with these examples, need simply uncomment the lines loading the `cali` and `mgcv` packages.
+
+## *Pf*PR~2-10~ matching using malariaEquilibrium
+
+The `malariaEquilibrium` package function `human_equilibrium()` returns the canonical equilibrium solution for a given EIR and the *Pf*PR~2-10~ can be calculated the output (see <https://github.com/mrc-ide/malariaEquilibrium> for more information). The first step is to generate a large range of EIR values to generate matching *Pf*PR~2-10~ values for. Next, we load the package's default parameter set (`Jamie_parameters.rds`), specify an effective clinical treatment coverage (`ft`) of 0.45 and, for each EIR value generated, run the `human_equilibrium()` function.
+
+The `human_equilibrium()` function returns a dataframe containing the proportion of each age class in each state variable at equilibrium. The *Pf*PR~2-10~ can be calculated from the output for each EIR value by summing the proportion of people aged 2-10 with cases of malaria (`pos_M`) and dividing this proportion by the proportion of the population between the ages 2-10 (`prop`). Finally, we store the matching EIR and *Pf*PR~2-10~ values in a data frame.
+
+```{r}
+
+# Establish a range of EIR values to generate matching PfPR2-10 values for:
+malEq_EIR <- seq(from = 0.1, to = 50, by = 0.5)
+
+# Load the base malariaSimulation parameter set:
+eq_simparams <- malariaEquilibrium::load_parameter_set("Jamie_parameters.rds")
```

Comment:
change to `q_simparams <- malariaEquilibrium::load_parameter_set()`

Reply:
Changed both instances of `eq_simparams` to `q_simparams`